packages/web3-eth-accounts/test/integration/account.test.ts

```diff
@@ -0,0 +1,214 @@
+﻿/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { TransactionFactory } from '@ethereumjs/tx';
+import { Address } from 'web3-types';
+import { isHexStrict } from 'web3-utils';
+import { Web3ValidatorError } from 'web3-validator';
+import {
+	create,
+	decrypt,
+	encrypt,
+	hashMessage,
+	privateKeyToAccount,
+	privateKeyToAddress,
+	recover,
+	recoverTransaction,
+	sign,
+	signTransaction,
+} from '../../src';
+import {
+	invalidDecryptData,
+	invalidEncryptData,
+	invalidKeyStore,
+	invalidPrivateKeytoAccountData,
+	invalidPrivateKeyToAddressData,
+	signatureRecoverData,
+	transactionsTestData,
+	validDecryptData,
+	validEncryptData,
+	validHashMessageData,
+	validPrivateKeytoAccountData,
+	validPrivateKeyToAddressData,
+} from '../fixtures/account';
+
+describe('accounts', () => {
```

Comment:
For accounts we discussed in backlog grooming call that these scenarios are covered in unit testing so we dnt need same scenarios in system tests.

Reply:
I've decided to run it in browsers/electron to check how it works. and found problem with firefox

Reply:
and also found problem with cypress .each method and fix it.